dcpquery/db/__main__.py

```diff
@@ -4,13 +4,14 @@
 
 import os, sys, argparse, logging, json
 
+import alembic
```

Comment:
alembic doesn't seem to be used in this module, is that right? Consider removing unused import.

Reply:
👍